examples/calendar/backend/services/users/controller.ts

```diff
@@ -0,0 +1,58 @@
+import { NextFunction, Request, Response } from "express";
+
+import { User } from "../../models/user.model";
+
+export const find = (req: Request, res: Response, next: NextFunction) => {
+  // If a query string ?publicAddress=... is given, then filter results
+  const whereClause =
+    req.query && req.query.publicAddress
+      ? {
+          where: { publicAddress: req.query.publicAddress },
+        }
+      : undefined;
+
+  return User.findAll(whereClause)
+    .then((users: User[]) => res.json(users))
+    .catch(next);
+};
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  // AccessToken payload is in req.user.payload, especially its `id` field
+  // UserId is the param in /users/:userId
+  // We only allow user accessing herself, i.e. require payload.id==userId
+  if ((req as any).user.payload.id !== +req.params.userId) {
```

Comment:
[placeholder] please check the previous comments.

Reply:
Removed comments and updated the code likewise.